src/main/java/org/example/service/CustomerInputService.java

```diff
@@ -17,18 +19,21 @@ public GenderType getGender(){
         }
         return getGender();
     }
-    public int getAge(){
-        System.out.print("Age: ");
-        int age = sc.nextInt();
-        if(age>=18){
-            return age;
+    public int getAge(String dateOfBirth){
+        int birthYear = Integer.parseInt(dateOfBirth.substring(0,4));
+        int birthMonth = Integer.parseInt(dateOfBirth.substring(5,7));
+        int currentMonth = d.getMonth()+1;
+        int age = currentYear - birthYear;
+        if(birthMonth>currentMonth){
+            age--;
```

Comment:
just a question,
as you have this condition on birth month, shouldn't we need this for day as well?